.gitignore

```diff
@@ -3,3 +3,5 @@ phpunit.xml
 vendor/
 .phpunit.cache/
 .php-cs-fixer.cache
+.phpunit.cache
+.idea
```

Comment:
Not a problem to include `.idea`, but I think this folder should be in everyone's global `.gitignore`